pkg/ibmvpc/analysis_output_test.go

```diff
@@ -460,6 +460,7 @@ var tests = []*commonvpc.VpcGeneralTest{
 		NoLbAbstract: true,
 		Format:       vpcmodel.HTML,
 	},
+	// todo: before merge - this fails sometimes in git. understand why and have fixed (seems in Haim's areas).
 	{
```

Comment:
its not relevant to this PR, I saw it one also on another PR.
better remove this comment, and open an issue

Reply:
ok. #847